src/libraries/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -29,7 +29,7 @@ internal static partial class NegotiateStreamPal
         // defined in winerror.h
         private const int NTE_FAIL = unchecked((int)0x80090020);
 
-        internal static string QueryContextClientSpecifiedSpn(SafeDeleteContext securityContext)
+        internal static string QueryContextClientSpecifiedSpn(SafeDeleteContext _ /*securityContext*/)
```

Comment:
How are you deciding when to use `_ /*name*/` vs using a `#pragma warning disable IDE0060`?

Reply:
No hard rule, I tried to use it when the #pragma looked noisy to me but no problem to change it